Content.Server/Bed/Cryostorage/CryostorageSystem.cs

```diff
@@ -185,6 +188,8 @@ public void HandleEnterCryostorage(Entity<CryostorageContainedComponent> ent, Ne
         if (!TryComp<CryostorageComponent>(cryostorageEnt, out var cryostorageComponent))
             return;
 
+        _containerSystemExtensions.RemoveEntitiesFromAllContainers<MindContainerComponent>(ent.Owner, ["body_root_part"]); //SS220-cryo-mobs-fix
```

Comment:
Ссылайся на константу
```suggestion
        _containerSystemExtensions.RemoveEntitiesFromAllContainers<MindContainerComponent>(ent.Owner, [SharedBodySystem.BodyRootContainerId]); //SS220-cryo-mobs-fix
```